IanGRap/Squish
# Random starting blocks

Contributors: Coulter, Austin

Goal:
- Implement a feature where the configuration of blocks and players changes every time you start a new match

Important Files Changed:
- MatchManager.cs 
    -Calls SceneCreator function to create the starting configuration / Calls PlayerColorController function to assign colors to the players
- Materials.cs 
    - static class that stores all potential colors of players
- PlayerColorController.cs
   - Assigns unique color to each player at the start of the match
- SceneCreator.cs
    - Chooses a configuration prefab, then instantiates player and block objects according to their locations in the configuration prefab
- SquishDetector.cs
    - Set the color of the particles according to the color of the player's material

How to Test:
- Load up the 2017 scene and make sure that you get different starting configurations each time you restart the match
    - This should work when you press Restart from the pause menu, Press Start to Play Again, and start the match from PlayerSelect
- Make sure that the functionality also works in a standalone build
